lib/gcp.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
```

Comment:
I would suggest we add a `lib/auth` directory to gather all the authentication implementations.

Reply:
I was going to that after exec support :+1: Done